<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carteras</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../logo.ico</normaloff>../../../logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background: rgb(160, 160, 160);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameBarra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonCartReal">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cartera Real</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../feather/dollar-sign.svg</normaloff>../../../feather/dollar-sign.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCartVirt">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cartera Virtual</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../feather/pie-chart.svg</normaloff>../../../feather/pie-chart.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddISIN">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AÃ±adir ISIN</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../feather/plus-circle.svg</normaloff>../../../feather/plus-circle.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLogout">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Log Out</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../feather/log-out.svg</normaloff>../../../feather/log-out.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameMain_Up">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>821</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="labelISIN">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>140</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic Medium</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ISINs en Cartera:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>207</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bienvenido de nuevo! </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelEmail">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>603</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS UI Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUsuario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listIsins">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>361</width>
        <height>151</height>
       </rect>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>68</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbModo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentText">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>190</y>
       <width>821</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout">
      <item>
       <widget class="QLabel" name="labelNoIsin">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
